packages/components/src/forms/form-label/index.stories.jsx

```diff
@@ -0,0 +1,19 @@
+import FormLabel from '.';
+
+export default { component: FormLabel, title: 'packages/components/Form Label' };
+
+const Template = ( args ) => {
+	return (
+		<form>
+			<FormLabel { ...args }>Button Label</FormLabel>
```

Comment:
Non essential.
Maybe add the `Button Label` as a dynamic arg.

Reply:
Hey thanks for chiming in Dehan -- I thought about doing so, but wasn't sure how to accomplish this without adding it to `Default.args` below. 

The issue with adding it to Default.args is that the label is then shown as a "Control", which I believe mistakenly implies that it might be a prop that FormLabel accepts.

<img width="1484" alt="Screenshot 2023-08-10 at 2 20 15 PM" src="https://github.com/Automattic/wp-calypso/assets/5414230/0efb7530-2edc-42d5-9971-0492924fe9bb">